### ISSUE TITLE
Client endpoint configuration '*' was not found in 0 endpoints.

### ISSUE THREAD

**Issue by maresb** [hirschmann/nbfc#194]
```
System:    Kernel: 4.4.0-64-generic x86_64 (64 bit) Desktop: Unity 7.4.0  Distro: Ubuntu 16.04 xenial
Machine:   System: ASUSTeK (portable) product: UX32VDA v: 1.0
           Mobo: ASUSTeK model: UX32VDA v: 1.0 Bios: American Megatrends v: UX32VDA.203 date: 01/28/2013
```

I can't seem to get nbfc running properly.  I install `mono-complete` from `http://download.mono-project.com/repo/debian wheezy main`.  Then I download `https://github.com/hirschmann/nbfc/archive/1.5.0.tar.gz` and run `build.sh`.  This compiles files to `nbfc-1.5.0/Core/X/bin/Release/`.  I'm trying to follow the instructions in `https://github.com/hirschmann/nbfc/wiki/First-steps`, but there is no `Linux/bin/ReleaseLinux/`, only `Linux/`.  I tried copying the contents of the various `nbfc-1.5.0/Core/X/bin/Release/` to `/opt/nbfc/` and I can start the service with `service nbfc start` but `/opt/nbfc/start-nbfcservice.sh` fails silently.  Whenever I run `/opt/nbfc/nbfc.exe X` where `X` is `start` `stop` or `config`, I get the error 
```
Client endpoint configuration '*' was not found in 0 endpoints.
```
I don't know what this means, and I suspect nbfc is not properly installed.  Any suggestions?  Thanks!!!

**Comment by maresb** [comment]
I resolved the problem by cloning the repository instead of downloading the .tar.gz.

**Comment by hirschmann** [comment]
```
Client endpoint configuration '*' was not found in 0 endpoints.
```
Sounds like the application configuration files are missing (.exe.config).
Anyway, since you've resolved this on your own, I'm closing this.

**Comment by maresb** [comment]
The archive https://github.com/hirschmann/nbfc/archive/1.5.0.tar.gz has a structure which is inconsistent with the documentation. Particularly, there is no ReleaseLinux directory. It seems to me like either deleting or recreating the archive would be helpful. (As far as I can tell, I followed instructions correctly and ended up with something which didn't work.)

**Comment by hirschmann** [comment]
I can't delete the archive. Github automatically generates it for every release.
The `ReleaseLinux` directory is only available if the build succeeded, so maybe something went wrong during the build.

**Comment by maresb** [comment]
Ah, I see. I was following your build instructions which say 
"If you aren't familiar with git you can download the latest revision of the source code as .zip file on github." Out of laziness I downloaded the archive assuming it would work, but it didn't. Let me know if there's anything I can do to help. Thanks for all your hard work!!!

**Comment by hirschmann** [comment]
Well, if you download the latest sources (master) it should build just fine. The problem is: I broke the linux build configuration with 1.4.5 beta and fixed it only after 1.5 (see #157).
I somehow forgot about this - my bad :/

**Comment by maresb** [comment]
Sorry, now I see specifically where I got it from.  I wanted the latest stable release, so I went to `https://github.com/hirschmann/nbfc/releases` and downloaded the file in question.  The file currently listed there doesn't work.  Hope that helps!
